tools/device_configuration_checker.py

```diff
@@ -299,35 +300,38 @@ def _run_tests_on_know_good_image(args, device):
 
     image = None
     if device.model.lower() == "beagleboneblack":
-        image = image_directory_path
+        image = "good-image.tar.bz2"
     elif device.model.lower() == "edison":
-        image = os.path.join(
-            image_directory_path,
-            "ostro-image-edison.ext4")
+       image = "good-image.ext4"
```

Comment:
Since we have a majority of PC-like devices currently, I though that using if/elif/else to basically spell out the exceptions in the initial branches and then to use the else branch as a catch-all for the PC-like devices would be easier.

Then again, it could silently if new non-pc-like devices are added, so there's that